std/os/bits/linux.zig

```diff
@@ -119,6 +119,23 @@ pub const O_RDONLY = 0o0;
 pub const O_WRONLY = 0o1;
 pub const O_RDWR = 0o2;
 
+pub const kernel_rwf = i32;
```

Comment:
This should be unsigned. There's a precedent set here. I understand that the corresponding type in the C API is signed, but according to the ABI there is no difference, and unsigned works better in Zig code, as well as making more sense in general. Note that the syscalls that use this signed type as is will be compile errors as they cast an i32 to a usize. However making this unsigned will fix that problem.